source/use-nunjucks/index.html.md.erb

```diff
@@ -0,0 +1,74 @@
+---
+title: Use Nunjucks
+weight: 45
+---
+
+# Use Nunjucks
+
+You can use our Nunjucks code to generate the HTML for pages and components.
+
+You can also change the HTML that's generated by passing options into the Nunjucks code.
+
+If you use Nunjucks, your project will be easier to update when a new version of GOV.UK Frontend is released.
+
+To use Nunjucks:
+
+- you must [install GOV.UK Frontend with Node.js package manager (npm)](/installing-with-npm/)
+- your application and any frameworks that support it must use Node.js
+
+You do not need to follow this guidance to [use Nunjucks macros in the GOV.UK Prototype Kit](https://design-system.service.gov.uk/get-started/prototyping/#using-nunjucks-macros).
+
+## Set up Nunjucks and use the page template
```

Comment:
I suspect in most cases, what the user does is going to look quite different to this, depending on what framework they're using. For example, if they're using express they might install [`express-nunjucks`](https://www.npmjs.com/package/express-nunjucks); if they're using eleventy it already has nunjucks installed and support built in.